src/Main.java

```diff
@@ -1,5 +1,17 @@
 public class Main {
     public static void main(String[] args) {
```

Comment:
As no comparison of these algorithms was added, as is required in the task, the mark is lowered for that as well.